autogen/safer-cluster/variables.tf.tmpl

```diff
@@ -68,7 +68,7 @@ variable "subnetwork" {
 variable "kubernetes_version" {
   type        = string
   description = "The Kubernetes version of the masters. If set to 'latest' it will pull latest available version in the selected region. The module enforces certain minimum versions to ensure that specific features are available. "
-  default     = "latest"
+  default     = ""
```

Comment:
```suggestion
  default     = null
```

Reply:
It seems like we want to avoid setting kubernetes_version at all by default?

Reply:
That makes sense.